quality.gradle

```diff
@@ -78,11 +78,13 @@ task pmd(type: Pmd, group: 'verification') {
     }
 }
 
-android {
-    lintOptions {
-        abortOnError true
-        xmlReport true
-        htmlReport true
-        lintConfig file("${project.rootDir}/AndroidCodeQualityConfig/quality/lint/lint.xml")
+afterEvaluate {
```

Comment:
Ya, but that should be used with applying this script in root project, I want keep these two ways both.